glimmer-version/app/templates/components/focusing-outlet.hbs

```diff
@@ -0,0 +1,3 @@
+{{#focusing-inner outletState=(-get-dynamic-var "outletState") focus=(action "checkFocus")}}
```

Comment:
Anybody who is working on this I'll make aware of the "glimmer as branding vs. glimmer as implementation" difference. That's a cost I'm willing to have and since it makes sense to me (and I'll likely be the one maintaining it). 😛